ansible/host_vars/slack.openobservatory.org/vars.yml

```diff
@@ -45,10 +44,8 @@ slack_irc_config: |
       "server": "irc.oftc.net",
       "token": "{{ slack_irc_token }}",
       "channelMapping": {
-        {% for channel in secret_channels %}
-        "{{ channel.slack }}": "{{ channel.irc }}",
-        {% endfor %}
         "#ooni": "#ooni",
+        "#ooni-dev": "#ooni-dev",
```

Comment:
I'm unsure if #ooni-bots should be bridged. I'd rather drop that channel as current code of the bridge does not relay Alertmanager messages to IRC at all.

Reply:
I think that is reasonable. It's mostly useful to us and we are using the IRC bridge anyways for other channels.

Less channels we have to maintain with broken support the better I think.